Sources/GRPCNIOTransportCore/Client/Connection/Connection.swift

```diff
@@ -421,6 +421,14 @@ extension Connection {
 
     private let http2Stream: NIOAsyncChannel<RPCResponsePart, RPCRequestPart>
 
+    var peerInfo: String {
```

Comment:
I think we should call this `remoteInfo` to mirror its client counterpart

Reply:
Yeah that works, I don't mind about the suffix so much. I'd like the consistency between "remote" and "local" (vs  "peer" and "local")